operational/itam/schedule_itam_report/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.1.2
+
+- Added a way to decode the pagination token
```

Comment:
This should be reworded in a way that would make sense to a non-technical person that wants to know what this change means for them. Explain the bug or problem they would have experienced that this fixes rather than describing it as you would to another engineer.

Reply:
The new version still has the same fundamental issue. "Added a filter to convert `%3D` to his corresponding `=` symbol to prevent bad api token request." still won't make much sense to a non-technical user.

Think of it from the user's point of view. What was the problem that they may have encountered before this change? Describe that problem in the terms that the user would understand, not an engineer, and then say you fixed that problem in the CHANGELOG.

Reply:
Hey @XOmniverse, @vadgun

I applied the changes to make the text more user friendly, we agreed on using the following:

Fixed a pagination bug that caused an error when the policy template requested information from the IT Asset Management (ITAM) server, thus failing the execution.